resources/dashboards/Analyze-Dashboard.json

```diff
@@ -1,11 +1,11 @@
-   {
-      "title": "Analyze Dashboard",
-      "hits": 0,
-      "description": "",
-      "panelsJSON": "[{\"col\":10,\"id\":\"Top-Applications-By-Bandwidth-(pie)\",\"row\":1,\"size_x\":3,\"size_y\":5,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Top-Applications-By-Bandwidth-(histogram)\",\"row\":1,\"size_x\":9,\"size_y\":5,\"type\":\"visualization\"},{\"id\":\"Analyze-Table\",\"type\":\"search\",\"size_x\":12,\"size_y\":4,\"col\":1,\"row\":6,\"columns\":[\"SrcIP\",\"DestIP\",\"Application\",\"Duration\",\"FlowCompleted\",\"Session\"],\"sort\":[\"TimeUpdated\",\"desc\"]}]",
-      "version": 1,
-      "timeRestore": false,
-      "kibanaSavedObjectMeta": {
-         "searchSourceJSON": "{\"filter\":[{\"query\":{\"query_string\":{\"analyze_wildcard\":true,\"query\":\"*\"}}}]}"
-      }
+{
+   "hits": 0, 
+   "timeRestore": false, 
+   "description": "", 
+   "title": "Analyze Dashboard", 
+   "panelsJSON": "[{\"col\":10,\"id\":\"Top-Applications-By-Bandwidth-(pie)\",\"row\":1,\"size_x\":3,\"size_y\":3,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Top-Applications-By-Bandwidth-(histogram)\",\"row\":1,\"size_x\":9,\"size_y\":6,\"type\":\"visualization\"},{\"id\":\"Analyze-Table\",\"type\":\"search\",\"size_x\":12,\"size_y\":4,\"col\":1,\"row\":7,\"columns\":[\"SrcIP\",\"DestIP\",\"Application\",\"Duration\",\"FlowCompleted\",\"Session\"],\"sort\":[\"TimeUpdated\",\"desc\"]},{\"id\":\"Top-Applications-By-Packet-Count\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":3,\"col\":10,\"row\":4}]", 
+   "version": 2, 
```

Comment:
Version should be 2 and panelsJSON has changed to add the new Top Apps by packet count chart.